client/app/components/dynamic-parameters/DynamicParameters.less

```
@@ -0,0 +1,36 @@
+@import '../../assets/less/inc/variables';
+
+.redash-datepicker {
+  width: auto !important;
+
+  .ant-calendar-picker-clear {
+    right: 35px;
+  }
+
+  &.date-range-input .ant-calendar-picker-input {
+    width: 376px;
+    transition: width 150ms ease-in-out;
```

I like this speed better.
Feel free to reject if you disagree.

```suggestion
    transition: width 100ms ease-in-out;
```